### PR TITLE
Pre-Processing: Resize #24

### PR DESCRIPTION
**Pull request by ouzibigouzi**

### CODE REVIEW

File: notebooks/EDA-and-modeling.ipynb [hunk 1]
```diff
@@ -16,412 +16,451 @@
...
-    }
-   },
-   "outputs": [],
-   "source": [
-    "# fillna with mean.. \n",
-    "X_train[\"altitude_low_meters\"] = X_train[\"altitude_low_meters\"].fillna(altitude_low_meters_mean)\n",
-    "X_train[\"altitude_high_meters\"] = X_train[\"altitude_high_meters\"].fillna(altitude_high_meters_mean)\n",
-    "X_train[\"altitude_mean_log\"] = X_train[\"altitude_mean_log\"].fillna(altitude_mean_log_mean)"
+    "# Specify the folder path containing the images\n",
+    "folder_path = \"/Users/ouzibigouziouzi/Desktop/FNNPK/-N\"\n",
```
Comment (LnawI) [inline notebooks/EDA-and-modeling.ipynb]: You want to keep it like this? Than when combining snippets, responsible person has to adjust the path.

**Review by LnawI — APPROVED** [review]

File: notebooks/EDA-and-modeling.ipynb (hunk 1 above)
Comment (ouzibigouzi) [reply notebooks/EDA-and-modeling.ipynb]: i would change it to '/path/to/input/images' to standarize it, what @richiwang and @Linn2202 think?

Comment (richiwang) [reply notebooks/EDA-and-modeling.ipynb]: that seems a good solution for now, I will search a way of not hardcoding the path, if there is one

**Comment by Linn2202** [comment]
I was at first a bit confused by the double important in the beginning. But nice, easy function to resize - thank you!

**Review by Linn2202 — APPROVED** [review]

**Pull request merged**